src/components/SettingsPanel.vue

```diff
@@ -16,9 +17,11 @@
 import { Vue, Component } from "vue-property-decorator";
 import AnnotationToggles from "@/components/AnnotationBrowser/AnnotationToggles.vue";
 import ViewerSettings from "@/components/ViewerSettings.vue";
+import UISettings from "@/components/UISettings.vue";
+import UserProfileSettings from "@/components/UserProfileSettings.vue";
 
-@Component({ components: { AnnotationToggles, ViewerSettings } })
+@Component({ components: { AnnotationToggles, ViewerSettings, UISettings, UserProfileSettings } })
```

Comment:
You can rename `UISettings` to `UiSettings` to avoid weird `<u-i-settings />` above
Also I think that you don't need to import `UserProfileSettings`